lib/bullet_train/action_models/scaffolders/performs_export_scaffolder.rb

```diff
@@ -4,6 +4,18 @@ module BulletTrain
   module ActionModels
     module Scaffolders
       class PerformsExportScaffolder < SuperScaffolding::Scaffolder
+        # TODO these methods were removed from the global scope in super scaffolding and moved to `Scaffolding::Transformer`,
+        # but this gem hasn't been updated yet.
+
+        def legacy_replace_in_file(file, before, after)
+          puts "Replacing in '#{file}'."
+          target_file_content = File.open(file).read
+          target_file_content.gsub!(before, after)
+          File.open(file, "w+") do |f|
+            f.write(target_file_content)
+          end
+        end
```

Comment:
Adding this method to all the classes that reference it seems to do the trick for those errors. There's probably a better way to handle this, but first I'm just trying to get things working.